Makefile

```diff
@@ -54,6 +54,10 @@ YAMLFMT          := $(TOOLS_BIN_DIR)/yamlfmt
 STRINGER         := $(TOOLS_BIN_DIR)/stringer
 WSL				 := $(TOOLS_BIN_DIR)/wsl
 K3D              := $(TOOLS_BIN_DIR)/k3d
+GENERATE_CONST   := $(TOOLS_BIN_DIR)/generate-const
```

Comment:
We do not only generate constants, but also generate the security scan config. Maybe call it POPULATE_IMAGES or something?

Reply:
good point, I renamed it to `POPULATE_IMAGES`